### ISSUE TITLE
Subjects from MEF are not displayed correctly in document detailed view

### ISSUE THREAD

**Issue by pronguen** [rero/rero-ils#2609]
**How it works**

MEF subject are not displayed in the same way than MEF contributors:
![image](https://user-images.githubusercontent.com/8154915/146752817-407e107f-1c97-45b0-8332-6c719694509f.png)

MEF subjects are not present in the subject facet:
![image](https://user-images.githubusercontent.com/8154915/146752942-0f2db305-4e90-4fa7-98f3-c1c611ed6c72.png)

**How it should works**

In the public and professional interfaces, *document detailed view* + *document subject facet*:
* Subjects from MEF are displayed in the same way than MEF contributors (same logic: multilingual, etc.)
    * They are clickable.
* Local subjects are displayed as follow
    * Field `preferred_name` for types `bf:Place`, `bf:Person`, `bf:Organization`
    * Field `title` / `creator` for type `bf:Work`
    * Field `term` for type `bf:Topic`, `bf:Temporal`
    * Ideally, the subjects are sorted by types for the display.

**Comment by ManaDeweerdt** [comment]
Duplicate with https://github.com/rero/rero-ils/issues/2354 ?

**Comment by PascalRepond** [comment]
@ManaDeweerdt Yes. I close the other one since this one is more detailed. Thank you for the heads-up.

**Comment by ManaDeweerdt** [comment]
> @ManaDeweerdt Yes. I close the other one since this one is more detailed. Thank you for the heads-up.

Maybe this one too? https://github.com/rero/rero-ils/issues/1869

**Comment by pronguen** [comment]
I reopen it because MEF persons are still not present in the subject facet.
